### PR TITLE
Fix non persisted state

### PR DESCRIPTION
**Pull request by HahaOO7**
Fix for [issue #78](https://github.com/joserodpt/RealScoreboard/issues/78)

### CODE REVIEW

File: realscoreboard-api/src/main/java/joserodpt/realscoreboard/api/scoreboard/RSBPlayer.java [hunk 1]
```diff
@@ -120,6 +120,7 @@ public void setRealScoreboardVisible(boolean realScoreboardVisible) {
             }
         }
         this.realScoreboardVisible = realScoreboardVisible;
+        RealScoreboardAPI.getInstance().getLogger().info("RealScoreboard visibility for " + p.getName() + " set to " + realScoreboardVisible);
```
Comment (neziw) [inline realscoreboard-api/src/main/java/joserodpt/realscoreboard/api/scoreboard/RSBPlayer.java]: This shouldn't be logged all time, switch to debugger or something other

**Comment by neziw** [comment]
@HahaOO7 have you tested these changes?

**Comment by HahaOO7** [comment]
Yes

**Comment by joserodpt** [comment]
Thank your for the pull request [HahaOO7](https://github.com/HahaOO7)!

**Pull request merged**